qiime/core/archiver.py

```diff
@@ -212,6 +218,20 @@ def _formatted_provenance(self):
                 'parameter-references': self.provenance.parameter_references
             }
 
+    def get_index_paths(self):
+        # TODO: this extracts the entire archive, even though that isn't
+        # necessary to get the index paths. Refactor to avoid this.
+        if not os.path.exists(self._data_dir):
+            self._extract_data()
+        result = {}
+        for path in os.listdir(self._data_dir):
+            if os.path.isfile(os.path.join(self._data_dir, path)):
+                filename = os.path.split(path)[1]
+                if filename.startswith('index.'):
+                    ext = os.path.splitext(filename)[1]
+                    result[ext[1:]] = os.path.join(self._data_dirname, path)
```

Comment:
That's true, but what would an interface do with that (e.g., if the cli identifies two ``index*html`` files, which does it display)? Maybe we should only treat ``index.extension`` as the index associated with ``extension``? (This code would need to be adapted to do that.)